packages/libs/client/src/client/client.ts

```diff
@@ -239,7 +252,7 @@ export interface ICreateClient {
    * Useful when you are working with a single network and chainId.
    * @param hostUrl - the URL to use in the client
    */
-  (hostUrl: string): IClient;
+  (hostUrl: string, defaults?: { confirmationDepth?: number }): IClient;
```

Comment:
Can we add some jsdoc for defaults to make it really clear what is to be expected when passing this value? I assume when you createClient with certain defaults it is applied to all applicable the functions returned

Reply:
I will add the docs. yes in this level it is applied to all applicable functions